src/components/GasOptions/tests/index.test.js

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import sinon from 'sinon';
+import BigNumber from 'bignumber.js';
+import {
+  shallow,
+} from 'enzyme';
+import {
+  intl,
+} from 'jest/__mocks__/react-intl';
+
+import GasOptions from '../index';
+
+describe('<GasOptions />', () => {
+  let props;
+  beforeEach(() => {
+    props = {
+      intl,
+      gasStatistics: {
+        safeLow: 19.0,
+        safeLowWait: 22.1,
+        fast: 100.0,
+        fastWait: 0.7,
+        fastest: 250.0,
+        fastestWait: 0.5,
+        average: 41.0,
+        avgWait: 1.6,
+        block_time: 13.98421052631579,
+        speed: 0.8085666916661166,
+        blockNum: 7068199,
+      },
+      defaultGasLimit: 800000,
+      defaultGasPrice: 10,
+    };
+  });
+  describe('defaultOption does not set to manual', () => {
+    let wrapper;
+    let gasPriceInput;
+    let onChangeSpy;
+    describe('when gasStatistics available', () => {
+      beforeEach(() => {
+        onChangeSpy = sinon.spy();
```

Comment:
In other parts of the app we've been using `jest` instead of `sinon` for things like this, any reason sinon works better in this scenario?